docs/UsingOmnisci/index.rst

```diff
@@ -0,0 +1,25 @@
+OmniSci
```

Comment:
Until we don't exclude Ray dependency out of the OmniSci engine, we should probably use the former in the names of the backend (`OmnisciOnRay`).

Reply:
No, we don't use Ray de facto so let's not mention it

Reply:
Ok but let's plan refactor for omnisci backend for one of the near release to get rid of Ray if we don't want to use it. Currently, it looks confusing.